docs/swarm/join-nodes.md

```diff
@@ -100,29 +105,29 @@ $ docker swarm join \
 This node joined a swarm as a manager.
 ```
 
-<!--TODO WIP
-Manager nodes use the listen address for cluster management communications. The
-other nodes on the swarm must be able to access the manager node on the
-IP address and port you specify for the listen address.
-
-Especially when there are multiple active network interfaces, you should
-you explicitly define the listen address when you add a manager node to the a
-swarm:
-
-  ```bash
-  docker swarm join \
-  --token <MANAGER-TOKEN> \
-  --listen-addr <NODE-IP>:<PORT> \
-  <MANAGER-IP>:<PORT>
-  ```
-this will change for https://github.com/docker/docker/pull/24237 ->>
-Replace <NODE-IP> with the IP address of the node that is joining the swarm.
-Replace <MANAGER-IP> with the address of the swarm manager.
-
-Only manager nodes use the listen address. If you specify `--listen-addr` for a
-worker node, the node only uses the listen address if it is promoted to a
-manager.
--->
+### Advertise address
+
+Manager nodes use an **advertise address** to allow other nodes in the swarm
+access to the API and overlay networking. If the system has multiple IP
+addresses, you must specify the correct  `--advertise-addr` to enable
+inter-manager communication and overlay networking:
```

Comment:
Another thing we should mention here is that when you set up the swarm with `swarm init`, you need to specify the `--advertise-addr` if the address where other nodes will use to reach the first node is not the same as the address it sees as its own. One example of where this comes into play is a cloud setup that spans different regions (or availability zones?). In a lot of cloud setups, hosts have internal addresses, which are the ones you see with `ifconfig`, and external addresses that you use from outside that zone or region. The first manager needs to be told about its external address with `--advertise-addr` so that it can spread that information to other nodes that will connect to it in the future.